### PR TITLE
[Android] Fix cut off frames in GIF, Keep frames in the same target orientation

### PR DESCRIPTION
**Pull request by mikepenz**
* fix bug causing cut off images in the GIF if the screen rotates during a UI test case
  * detect target orientation based on the first frame
  * rotate follow up frames according to the initial frames detected orientation

### CODE REVIEW

**Comment by mikepenz** [comment]
@Malinskiy any suggestion what might be wrong with the labeler ?

**Comment by BorisOsipov** [comment]
@mikepenz about labeler see https://github.com/Malinskiy/marathon/pull/301#issuecomment-566355128

**Comment by mikepenz** [comment]
@BorisOsipov thank you very much

File: vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/screenshot/ScreenCapturer.kt [hunk 1]
```diff
@@ -31,9 +31,16 @@ class ScreenCapturer(
     suspend fun start() = coroutineScope {
         val outputStream = FileImageOutputStream(fileManager.createFile(FileType.SCREENSHOT, poolId, device.toDeviceInfo(), test))
         val writer = GifSequenceWriter(outputStream, TYPE_INT_ARGB, DELAY, true)
+        var targetOrientation = PORTRAIT
         while (isActive) {
             val capturingTimeMillis = measureTimeMillis {
-                getScreenshot()?.let { writer.writeToSequence(it) }
+                getScreenshot(targetOrientation)?.let {
+                    if (targetOrientation == UNDEFINED) {
```
Comment (Vacxe) [inline vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/screenshot/ScreenCapturer.kt:38]: Look like that this condition will be always ```false```.
Maybe you wanted to define ```var targetOrientation = PORTRAIT``` to ```UNDEFINED``` ?

Comment (mikepenz) [reply vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/screenshot/ScreenCapturer.kt:38]: Great catch. It should be actually UNDEFINED as long as it is unknown which orientation the images will have. based on the first screenshot we retrieve

**Review by Vacxe — APPROVED** [review]

**Comment by Malinskiy** [comment]
For the future, we might consider recording just a square gif if we need to support cases with changing rotation better.

**Pull request merged**